### PR TITLE
Bring the font size down to 14px

### PR DESCRIPTION
**Pull request by marcoambrosini**
Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

**Review by piperek — APPROVED** [review]
I like it! It's a good improvement as discussed earlier.

**Comment by georgehrke** [comment]
It looks better now 👍

But what about all the other places where we use px-based font-sizes?

e.g. h3 is just set to 16px, which looks small now, as the normal text is 14px.

**Comment by gary-kim** [comment]
I thought the font size was just increased to 15px. Why back down to 14px?

**Comment by skjnldsv** [comment]
> I thought the font size was just increased to 15px. Why back down to 14px?

a bit too high :)

File: core/css/styles.scss [hunk 1]
```diff
@@ -89,7 +89,7 @@ body {
 	background-color: var(--color-main-background);
 	font-weight: normal;
 	/* bring the default font size up to 15px */
```
Comment (gary-kim) [inline core/css/styles.scss]: Comment is no longer accurate. Could it be changed?

Comment (ma12-co) [reply core/css/styles.scss]: :man_facepalming: 
sure

**Review by gary-kim — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**